### PR TITLE
chore: Move to app-sre

### PR DESCRIPTION
**Pull request by yzainee**
files added to support movement to app-sre cd

### CODE REVIEW

File: Dockerfile.app-sre [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+LABEL name="bayesian-data-importer" \
+      description="Data model and api services." \
+      email-ids="yzainee@redhat.com" \
+      git-url="https://github.com/fabric8-analytics/fabric8-analytics-data-model" \
+      target-file="Dockerfile" \
+      app-license="GPL-3.0"
+
+RUN microdnf install python3 git && microdnf clean all
+RUN pip3 install --upgrade pip --no-cache-dir
```
Comment (arajkumar) [inline Dockerfile.app-sre:11]: You need to repeat `--no-cache-dir` whereever you do `pip install`

File: Dockerfile.app-sre [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-utils.git@6cb0a2d
```
Comment (arajkumar) [inline Dockerfile.app-sre:12]: Didn't we streamlined depenedency installation for data-model?
CC: @sawood14012

File: build_deploy.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+QUAY_IMAGE="quay.io/app-sre/${BASE_IMG}"
+IMG="${BASE_IMG}:latest"
+
+GIT_HASH=`git rev-parse --short=7 HEAD`
+
+# build the image
+docker build  --no-cache \
+              --force-rm \
+              -t ${IMG}  \
+              -f ./Dockerfile .
```
Comment (arajkumar) [inline build_deploy.sh:14]: use Dockerfile.app-sre here
```suggestion
              -f ./Dockerfile.app-sre .
```

File: openshift/template.app-sre.yaml [hunk 4]
```diff
@@ -0,0 +1,234 @@
+apiVersion: v1
```
Comment (arajkumar) [inline openshift/template.app-sre.yaml:1]: This is not needed incase we are using UBI image, you can use existing template.

**Review by arajkumar — CHANGES_REQUESTED** [review]

File: Dockerfile.app-sre (hunk 2 above)
Comment (sawood14012) [reply Dockerfile.app-sre:12]: we might need to upgrade pip , @yzainee you can use `pip3 install -r requirements.txt ` or for test requirements `pip3 install -r requirements.txt -r test/requirements.txt` no need to explicitly install utils its already present in the requirements file

File: build_deploy.sh (hunk 3 above)
Comment (yzainee) [reply build_deploy.sh:14]: typo. rectified

File: openshift/template.app-sre.yaml (hunk 4 above)
Comment (yzainee) [reply openshift/template.app-sre.yaml:1]: i have checked in api server: https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/openshift/template.yaml Please cross check

File: Dockerfile.app-sre (hunk 1 above)
Comment (yzainee) [reply Dockerfile.app-sre:11]: done

File: openshift/template.app-sre.yaml (hunk 4 above)
Comment (arajkumar) [reply openshift/template.app-sre.yaml:1]: No, not required. I don't know why @rafiu007 did that.

Comment (rafiu007) [reply openshift/template.app-sre.yaml:1]: @arajkumar  did not make any changes in the openshift /api-server template rekated to app-sre switch.

Comment (yzainee) [reply openshift/template.app-sre.yaml:1]: @arajkumar you have any sample which is running properly without that?

Comment (arajkumar) [reply openshift/template.app-sre.yaml:1]: I'm not sure whether I understand your question, but the existing template.yaml will work. Don't add this.

Comment (yzainee) [reply openshift/template.app-sre.yaml:1]: ok so it will pick the existing file itself. no need to add this new file then?

Comment (yzainee) [reply openshift/template.app-sre.yaml]: file removed

File: Dockerfile.app-sre (hunk 2 above)
Comment (yzainee) [reply Dockerfile.app-sre]: done

**Review by arajkumar — APPROVED** [review]
> ADD ./ /tmp/f8a_data_model/
> ..
> RUN cd /tmp/f8a_data_model && pip3 install . --no-cache-dir

Though I've some concerns regarding the above step, it shall be fixed in a separate PR as I see similar pattern in existing Dockerfile as well.

lgtm

**Pull request merged**